docs/kubectl-port-forward.md

```diff
@@ -0,0 +1,65 @@
+## kubectl port-forward
+
+Forward 1 or more local ports to a container.
+
+### Synopsis
+
+Forward 1 or more local ports to a container.
+Examples:
+  $ kubectl port-forward -p mypod -c mycontainer 5000 6000
```

Comment:
Ok, I'll see if I can do this tonight, but it might have to wait until tomorrow.